java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -325,8 +325,8 @@ CompletableFuture<Long> linsert(
     CompletableFuture<String[]> brpop(String[] keys, double timeout);
 
     /**
-     * Inserts specified values at the tail of the <code>list</code>, only if <code>key</code> already
-     * exists and holds a list.
+     * If <code>key</code> is a list, inserts all the specified values at the tail of the list. If
+     * <code>key</code> is not a list, this performs no operation.
```

Comment:
I thought the better option was

Inserts all the specified values at the head of the list stored at `key`, only if the list already exists.

Reply:
That doesn't specify what happens when the list doesn't exist.